### DISCUSSION TITLE
No multi-site on System Messages.

### DISCUSSION THREAD

**Discussion by internetztube** [craftcms/cms#8605] [Ideas]
### Description

Is there a way to differentiate the site in "System Messages"-Emails? Instead of a differentiation between languages, I would need a dropdown with all sites. 

<img width="594" alt="bildschirmfoto 2018-11-05 um 16 05 45" src="https://user-images.githubusercontent.com/6382364/48005838-c2e23400-e114-11e8-9818-60f904ef4b08.png">

### Additional info

- Craft version: Craft Pro 3.0.27
- PHP version: 7.1.23
- Database driver & version: MySQL 5.7.24

**Comment by brandonkelly** [comment]
Currently those messages are saved per-language, not per-site. I’ll consider this a feature request.
